_documentation/subscriptions.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Subscriptions
+layout: doc
+level: "1"
+group: "subscription"
```

Comment:
The page does not show its children because of this typo:

```suggestion
group: "subscriptions"
```

You can test the changes by clicking "Details" on the Netlify check:

![grafik](https://user-images.githubusercontent.com/5811634/131242577-ebe1ab29-733a-41e1-be50-3f734b9ebe48.png)